app/components/List/tests/index.test.js

```diff
@@ -1,15 +1,22 @@
 import expect from 'expect';
-import { render } from 'enzyme';
+import { mount } from 'enzyme';
 import React from 'react';
 
 import List from '../index';
 import ListItem from 'components/ListItem';
 
 describe('<List />', () => {
   it('should render the component if no items are passed', () => {
-    const renderedComponent = render(
+    const renderedComponent = mount(
       <List component={ListItem} />
     );
+    expect(renderedComponent.find(List)).toExist();
+  });
+
+  it('should not render the items if no items are passed', () => {
```

Comment:
I feel like "expect renderedComponent => ListItem to exist" does not prove "It should not render the items if no items are passed"

Might it not make more sense to check for length/count of the list items to prove this test?
If no items are passed in (empty array) then we would expect to find 0 listItem components.

I might be missing something here though - if so I apologise in advance! :D